applications/migrate-fastify-app-to-platformatic-service/02-after-with-platformatic-service/test/routes.test.js

```diff
@@ -0,0 +1,50 @@
+import { test } from 'node:test'
+import assert from 'node:assert/strict'
+
+import { buildServer } from '@platformatic/service'
+
+import serviceConfig from '../platformatic.service.json' assert { type: 'json' }
+
+test('Basic API', async (t) => {
+  // const app = await buildServer('./platformatic.service.json')
+
+  // TODO: Is there a better way to turn off logging?
+  //       Node.js also emits a ExperimentalWarning message due to
+  //       the use of import assertions above.
+  serviceConfig.server.logger = false
```

Comment:
@mcollina As mentioned in this comment, is there a better way to turn off Fastify server logging in tests?

Reply:
no, that is it. Why?

Reply:
Ok, that's fine. Wanted to make sure this guide presents the simplest way to do it.

The real shame is that I'm turning off logging here, but then get two big warning messages emitted by Node.js when I run the tests because I'm using ESM + import assertions.

Reply:
Not ideal, but `node --no-warnings --test --test-reporter=spec test/*.test.js` removes the warning message noise when running the tests.